2.train-VAE/utils.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
I don't think you need this file. Please make sure to review the files that you're adding

Reply:
got it! I made the mistake of copying all the .py files into the module doing the trial and error method when trying to fix the beta VAE notebook.